### PR TITLE
feat: added option to change inkColor and additional tests for the same

### PR DESCRIPTION
**Pull request by SuyashSonawane**
As mentioned in the issue #46, I have added the option to pass color for the ink of document, below are some example of the same

image
image

I also ran and added tests for these functions

  Testing main function
    ✔ input text to pdf with no modification
    ✔ input text to pdf with ruled
    ✔ input text to pdf with custom ink color

  3 passing (269ms)
 
Please review the changes and provide any feedbacks 🙌

### CODE REVIEW

**Comment by SuyashSonawane** [comment]
while running npm build, I faced some errors regarding jimp, I think it has some issue with browsify

**Comment by SuyashSonawane** [comment]
Hi @alias-rahil, I fixed that issue, now 1 additional COLOR object is available under **handwritten.js/constants**, also I added the inkColor change option in the web-demo https://alias-rahil.github.io/handwritten.js/

image

File: src/index.js [hunk 1]
```diff
@@ -4,13 +4,24 @@ const Jimp = require('jimp')
...
-      .split('').concat(['margin'])
-const jimpObjectPromises = []
-for (let i = 0; i < symbols.length; i += 1) {
-  for (let j = 0; j < 6; j += 1) {
-    dataset[i][j] = Buffer.from(dataset[i][j])
-    jimpObjectPromises.push(Jimp.read(dataset[i][j]))
+  ' !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~'
+    .split('').concat(['margin'])
+
+const COLORS = {
```
Comment (alias-rahil) [inline src/index.js:10]: `COLORS` is already defined in `constants.js`, import it from there. Remove this.

File: src/index.js [hunk 2]
```diff
@@ -117,24 +128,28 @@ function checkArgType (rawText, optionalArgs) {
     .ruled) !== 'undefined') {
     return false
   }
+  if (typeof (optionalArgs.inkColor) !== 'string' && typeof (optionalArgs
```
Comment (alias-rahil) [inline src/index.js:131]: (Optional) This manual input validation is getting very complicated now, Can you rewrite this whole part using [Joi](https://www.npmjs.com/package/joi)?

**Review by alias-rahil — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (alias-rahil) [reply src/index.js:131]: This is optional, I can instead create a new issue for this for someone else to pick up

Comment (SuyashSonawane) [reply src/index.js:131]: yes, I also thought that keeping track of args is too difficult now, I'll try to incorporate your suggestion.

**Comment by SuyashSonawane** [comment]
@alias-rahil performed suggested changes

**Review by alias-rahil — APPROVED** [review]
LGTM, thanks for your contribution :)

**Comment by alias-rahil** [comment]
One last change, can you bump the version to create a minor release? Then we can merge the PR

**Comment by SuyashSonawane** [comment]
sure

**Comment by alias-rahil** [comment]
Almost forgot about it, you'll need to update README and bin.js too

**Comment by SuyashSonawane** [comment]
> Almost forgot about it, you'll need to update README and bin.js too

yup no issue

1. run npm run build right 
2. increase version in package.json
3. add doc for new feature
4. add my name in readme

**Comment by alias-rahil** [comment]
> > Almost forgot about it, you'll need to update README and bin.js too
> 
> yup no issue
> 
> 1. run npm run build right
> 2. increase version in package.json
> 3. add doc for new feature
> 4. add my name in readme

Yes

**Comment by SuyashSonawane** [comment]
All set 💯

**Review by alias-rahil — APPROVED** [review]

**Comment by alias-rahil** [comment]
Thanks again @SuyashSonawane

**Comment by SuyashSonawane** [comment]
> Thanks again @SuyashSonawane

Thanks a lot for giving me the opportunity and taking out time to review my contribution.

**Pull request merged**